IotcRestApi/regulator_draft.md

```diff
@@ -0,0 +1,145 @@
+# API
+Bude se jednat o REST API, bude k dispozici Swagger.
+
+# Zabezpečení
+Komunikace bude zabezpečena bearer tokenem v hlavičce, informace budou doplněny.
+
+Většina endpointů přijímajících data v těle requestu vyžaduje položku `requestId`. Jedná se o jednoznačný identifikátor requestu, který generuje partner (jedná se o GUID) a slouží pro zajištění idempotence, čímž je docíleno toho, že akce je provedena jen jednou pokud dojde o opakovanému zavolání v určitém časovém intervalu.
```

Comment:
Neni to GUID ale UUID. GUID je pouze v .netu. https://www.ietf.org/rfc/rfc4122.txt


(Asi by to chtelo zmenit vsude v dokumentaci)

Reply:
ok